src/zoo.ml

```diff
@@ -197,7 +196,8 @@ struct
         | None -> fatal_error "I am sorry but this language has no interactive toplevel."
       in
       Format.printf "%s -- programming languages zoo@\n" L.name ;
-      Format.printf "Type %s to exit@." eof ;
+      Format.printf "Type %s to exit@\n" eof ;
```

Comment:
Would it be better to remove the period, because `split long lines with \.` can be confusing?